frontend/public/module/k8s/k8s.ts

```diff
@@ -38,8 +39,53 @@ export const groupVersionFor = (apiVersion: string) => ({
   version: apiVersion.split('/').length === 2 ? apiVersion.split('/')[1] : apiVersion,
 });
 
+const parseAPIVersion = (version: string) => {
+  const parsed = /^v(\d+)(?:(alpha|beta)(\d+))?$/.exec(version);
+  return parsed
+    ? { majorVersion: Number(parsed[1]), qualifier: parsed[2], minorVersion: Number(parsed[3]) }
+    : null;
+};
+
+export const crdVersionSort = (crdVersions: CRDVersion[]) => {
+  return crdVersions?.sort((v1, v2) => {
+    const v1Parsed = parseAPIVersion(v1.name);
+    const v2Parsed = parseAPIVersion(v2.name);
```

Comment:
You need to handle null values first. There's a runtime error if either is null.